### DISCUSSION TITLE
Can I download the program using Pybricks Code via USB cable?

### DISCUSSION THREAD

**Discussion by kubotafamily** [pybricks/support#1753] [Q&A] [answered]
I am writing a code to compete in WRO and the WRO rules prohibit Bluetooth connectivity.
https://github.com/orgs/pybricks/discussions/709
This discussion says that you cannot download the program with a USB cable, but is it still possible to download the program with Pybricks Code with a USB cable?
I would appreciate it if you could tell me.

**Comment by laurensvalk** [comment, marked as answer]
USB is not supported in Pybricks.

As far as I know, the global WRO rules state that Bluetooth is allowed when USB is not possible.

However, local competitions may enforce stricter rules, so be sure to check with your local organizers.

**Reply by kubotafamily** [reply]
Thank you for your answer.

I used Pybricks at the WRO Japanese final. Pybricks was awesome!

Unfortunately, I won't be able to compete in the world final this time, but I'll continue to use Pybricks in the future!

I'll do my best to compete in the world final next year!

**Reply by laurensvalk** [reply]
Thank you!
